docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md

```diff
@@ -69,6 +69,20 @@
 * Updated the Repository and Cart to display Sample Sheet and Metadata downloads as separate buttons for better visibility. <!--PEAR-2232-->
 * Minor text and styling improvements. <!--PEAR-452/2298/2341/2323/2253/2290-->
 
+* __Single-Cell RNAseq__:
+    * Visualize single-cell RNA-Seq data with cluster plots and gene expression overlays.
+* __OncoMatrix__:
+    * By default hide “splice region” consequence mutations.
+    * Support 5-category CNV; in OncoPrint mode, plot a white border around SSM to better distinguish it from CNV when columns are wide enough
```

Comment:
```suggestion
    * Support 5-category CNV: in OncoPrint mode, plot a white border around SSM to better distinguish it from CNV when columns are wide enough
```
A colon seems more appropriate here, but let me know if that wasn't your intention. It's probably cool either way.